Builder.cs

```diff
@@ -386,6 +386,15 @@ public async static Task<bool> FillFromId(RedumpHttpClient wc, SubmissionInfo in
                 info.DumpersAndStatus.Dumpers = [.. tempDumpers];
             }
 
+            // PS3 DiscKey
+            match = Constants.PS3DiscKey.Match(discData);
+            if (match.Success && info.Extras != null && string.IsNullOrEmpty(info.Extras.DiscKey))
+            {
+                // Validate key is not NULL
+                if (match.Groups[1].Value != null && match.Groups[1].Value != "<span class=\"null\">NULL</span>")
+                    info.Extras.DiscKey = match.Groups[1].Value;
+            }
```

Comment:
Minor thing to make it act like the rest of the statements:
```suggestion
            if (string.IsNullOrEmpty(info.Extras!.DiscKey))
            {
                // Validate key is not NULL
                match = Constants.PS3DiscKey.Match(discData);
                if (match.Success && match.Groups[1].Value != "<span class=\"null\">NULL</span>")
                    info.Extras.DiscKey = match.Groups[1].Value;
            }
```